components/counter-bar/README.md

```diff
@@ -11,120 +11,102 @@ CounterBar
 
 Simple
 ```jsx
-<CounterBar
- listTitle="All counters"
- name="name"
- counters={[
-   { id: 'abc', name: 'Counter 1', score: 0 },
-   { id: 'def', name: 'Counter 2', score: 2 },
-   { id: 'ghi', name: 'Counter 3', score: 2 },
-   { id: 'jkl', name: 'Counter 4', score: 9 },
-   { id: 'mno', name: 'Counter 5', score: 2 },
-   { id: 'pqr', name: 'Counter 6', score: 1 },
-   { id: 'stu', name: 'Counter 7', score: 0 },
- ]}
-/>
+<CounterBar>
+  <CounterBar.Total score={16}>All counters</CounterBar.Total>
+  <CounterBar.Container>
+    <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
+    <CounterBar.Item score={2}>Counter 2</CounterBar.Item>
+    <CounterBar.Item score={2}>Counter 3</CounterBar.Item>
+    <CounterBar.Item score={9}>Counter 4</CounterBar.Item>
+    <CounterBar.Item score={2}>Counter 5</CounterBar.Item>
+    <CounterBar.Item score={1}>Counter 6</CounterBar.Item>
+    <CounterBar.Item score={0}>Counter 7</CounterBar.Item>
+  </CounterBar.Container>
+</CounterBar>
 ```
 Active counter
 ```jsx
-<CounterBar
- listTitle="All counters"
- name="name"
- counters={[
-   { id: 'abc', name: 'Counter 1', score: 0 },
-   { active: true, id: 'def', name: 'Counter 2', score: 2 },
-   { id: 'ghi', name: 'Counter 3', score: 2 },
- ]}
-/>
+<CounterBar>
+ <CounterBar.Total score={5}>All counters</CounterBar.Total>
+ <CounterBar.Container>
+ <CounterBar.Item score={0}>Counter 1</CounterBar.Item>
```

Comment:
aha - this is the source of the dodgy indentation :)

Reply:
Ah yes - could be! I'll fix that across the board...